components/net_connect/README.md

```diff
@@ -0,0 +1,66 @@
+# net_connect
```

Comment:
Should we also consider design for testablity? Suggest adding a simple unit test, just the see how the current design is modular/testable

Reply:
Internal restructuring to improve mockability and testability is planned for a future release. I am considering a Hierarchical State Machine–based architecture.